docs/content/manual/release-specific/v1.7.0/test-rwx-fast-failover.md

```diff
@@ -0,0 +1,89 @@
+---
+title: RWX Fast Failover
+---
+
+## Related issues
+
+- https://github.com/longhorn/longhorn/issues/6205
+
+## LEP
+
+- https://github.com/longhorn/longhorn/pull/9069
+
+## Test step
+
+### Scale Test
+**Given** Longhorn cluster with 3 worker nodes.
+
```

Comment:
Before running the script to create multiple deployments, we need to set up a `StorageClass`. However, I'm not sure if you want to use a `hard` mount, a `soft` mount, or `both`. [In my previous tests](https://github.com/longhorn/longhorn/issues/6205#issuecomment-2257575626), I used a soft mount for the scale test case.

```suggestion

**And** Deploy an RWX volume with the following storage class settings (nfsOptions: "hard,timeo=50,retrans=1")

```yaml
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: longhorn-test
provisioner: driver.longhorn.io
allowVolumeExpansion: true
reclaimPolicy: Delete
volumeBindingMode: Immediate
parameters:
  numberOfReplicas: "3"
  staleReplicaTimeout: "2880"
  fromBackup: ""
  fsType: "ext4"
  nfsOptions: "hard,timeo=50,retrans=1"
```
**Or**

**And** Deploy an RWX volume with the following storage class settings (nfsOptions: "soft,timeo=250,retrans=5")
```yaml
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: longhorn-test
provisioner: driver.longhorn.io
allowVolumeExpansion: true
reclaimPolicy: Delete
volumeBindingMode: Immediate
parameters:
  numberOfReplicas: "3"
  staleReplicaTimeout: "2880"
  fromBackup: ""
  fsType: "ext4"
  nfsOptions: "soft,timeo=250,retrans=5"
```